src/sample.ceph.conf

```diff
@@ -1,14 +1,16 @@
 ##
 # Sample ceph ceph.conf file.
 ##
-# This file defines cluster membership, the various locations
-# that Ceph stores data, and any other runtime options.
-
-# If a 'host' is defined for a daemon, the init.d start/stop script will
-# verify that it matches the hostname (or else ignore it).  If it is
-# not defined, it is assumed that the daemon is intended to start on
-# the current host (e.g., in a setup with a startup.conf on each
-# node).
+# This file used to define cluster membership, the various locations
+# that Ceph stores data, and any other runtime options.  With the introduction
```

Comment:
s/any other/all other/

What do you think?

Reply:
I pushed a condensation